src/utils/kvt.py

```diff
@@ -0,0 +1,159 @@
+
+
+# lifted from:
+# - https://stackoverflow.com/questions/38987
+def merge_dicts(*dict_args):
```

Comment:
@taliaga this approach could be interesting (in general) for the creation of replies (if i understood the purpose). eventhough, i am not sure how much "manipulation" you are running on the json snippets.

Reply:
yeah, you've hit the nail on the head. I'm not sure either it's worth our while unless we have to fiddle heavily with the json which I don't think we do at the moment. I was just testing the waters, we can ditch it if it doesn't turn out to be useful.